Include info about running plugins with scheduled jobs in HA (#909)
mattermost/mattermost-developer-documentation: site/content/extend/plugins/server/ha.md

The proper way to deal with this case would be for the plugin to store the trigger word in a global store, such as the KV store. Then any time a user posts the plugin can pull the trigger word from the store and properly alert the user, regardless of which app server handles the request.

## Running a scheduled job in High Availability mode

> Using the [mattermost-plugin-api/cluster](https://github.com/mattermost/mattermost-plugin-api/blob/37eccf0f0f3e9e0737d017b555ef3e2e720b3021/cluster/job.go#L112) package, we can schedule jobs to perform background activity on a regular interval, without having to explicitly coordinate with other instances of the same plugin. Here's an example from the [Demo Plugin](https://github.com/mattermost/mattermost-plugin-demo/blob/d647f1ed7fdc384f5bc163a6bba689ab4293704e/server/activate_hooks.go#L72):

Reviewer: ```suggestion
Using the [mattermost-plugin-api/cluster](https://github.com/mattermost/mattermost-plugin-api/blob/37eccf0f0f3e9e0737d017b555ef3e2e720b3021/cluster/job.go#L112) package, we can schedule jobs to perform background activity at regular intervals, without having to explicitly coordinate with other instances of the same plugin. Here's an example from the [Demo Plugin](https://github.com/mattermost/mattermost-plugin-demo/blob/d647f1ed7fdc384f5bc163a6bba689ab4293704e/server/activate_hooks.go#L72):
```
